test/testdata/googleapis_test/gapic/packaging/api_defaults.yaml

```diff
@@ -19,6 +19,9 @@ author: Google Inc
 email: googleapis-packages@google.com
 homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
+development_status:
```

Comment:
We already have a `release_level` concept in gapic config - this seems like the same concept.